easybuild/tools/module_generator.py

```diff
@@ -539,12 +539,40 @@ def _generate_help_text(self):
                 else:
                     lines.append(" - %s contact: %s" % (contacts_type.capitalize(), contacts))
 
+        # Multi deps (if any)
+        multi_deps = self._generate_multi_deps_list()
+        section_txt = "This module is compatible with the following modules, one of each is required"
```

Comment:
the section name should be shorter imho, see https://github.com/akesandgren/easybuild-framework/pull/13